scipy/spatial/transform/rotation.pyx

```diff
@@ -529,6 +529,14 @@ cdef class Rotation(object):
         else:
             self._quat = quat.copy() if copy else quat
 
+    def __getstate__(self):
+        return np.asarray(self._quat, dtype=float), self._single
```

Comment:
Can we simply do
`return self._quat, self._single`?

`self._quat` is guaranteed to be a float array.

Reply:
https://github.com/scipy/scipy/pull/13425/files#diff-63d0677bac7b4878bcd481ed514bf9cdcf9c377eaa67b38b2942eb35a2b2a9cfL503

Yup, it's a typed memoryview. Since I'm not sure it has a .copy() attribute, it's easier to just asarray IMO.

Reply:
They do have a copy method (https://cython.readthedocs.io/en/latest/src/userguide/memoryviews.html#copying), but it doesn't matter much. This definitely works, and I have no experience with picklability of a memoryview. So I agree with leaving as is.